src/Components/AppDownloadBanner.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, Clickable, Text } from "@artsy/palette"
+import { FC, useEffect } from "react"
+import ChevronSmallRightIcon from "@artsy/icons/ChevronSmallRightIcon"
+import { useCursor } from "use-cursor"
+
+const TEXTS = [
+  "Get the app, get the art.",
+  "Get the app, and find the art you love.",
+  "Get the app, and artworks tailored to you.",
+  "Get the app, and discover works our curators love.",
+  "Get the app, and keep up with your favorite artists",
+  "Get the app, and be alerted about new artworks.",
+  "Get the app, and stay connected with galleries.",
+  "Get the app, and build your art world profile.",
+  "Get the app, and keep track of your collection.",
+  "Get the app, and stay informed on your artists’ markets.",
```

Comment:
I assume the long phrases are working fine within mobile view, is that right?

Reply:
> Get the app, and stay informed on your artists’ markets.

Breaks on "markets" — I might be able to tighten up the tracking on that one but for now just wanted to get something up so Ryan can confirm the timings.